packages/vuetify/src/composables/border-radius.ts

```diff
@@ -0,0 +1,61 @@
+// Utilities
+import { computed } from 'vue'
+import propsFactory from '@/util/propsFactory'
+
+// Types
+import type { PropType } from 'vue'
+
+export type BorderRadiusProp = boolean | 0 | '0' | '' | 'xs' | 'sm' | 'lg' | 'xl' | 'circle' | 'pill' | 'shaped' | 'tile' | null
+
+export interface BorderRadiusProps {
+  rounded?: BorderRadiusProp
+}
+
+const radius: BorderRadiusProp[] = [
+  true,
+  false,
+  0,
+  '0',
+  'xs',
+  'sm',
+  'lg',
+  'xl',
+  'circle',
+  'pill',
+  'shaped',
+  'tile',
+]
+
+export const makeBorderRadiusProps = propsFactory({
+  rounded: {
+    type: [Boolean, Number, String] as PropType<BorderRadiusProp>,
+    validator: (v: any) => radius.includes(v),
+  },
+})
+
+// Composables
+export function useBorderRadius (props: BorderRadiusProps) {
+  const borderRadiusClasses = computed(() => {
+    const classes = []
+    const rounded = props.rounded
+
+    /* istanbul ignore else */
+    if (rounded == null) {
+      // noop
+    } else if (rounded === true || rounded === '') {
+      classes.push('rounded')
+    } else if ([false, 0, '0', 'tile'].includes(rounded)) {
+      classes.push('rounded-0')
+    } else if (typeof rounded === 'string') {
+      const values = rounded.split(' ')
+
+      for (const value of values) {
+        classes.push(`rounded-${value}`)
+      }
+    }
+
+    return classes
+  })
```

Comment:
OK, we're going to *mingle together* the behavior AND *the Vue bindings* (e.g. `computed(() => )`)

[I wanted to make them separate so **testing wouldn't involve Vue at all**](https://github.com/vuetifyjs/vuetify/pull/12801/files#r553057616), but that's my personal taste.

Reply:
Personally I see no real benefit for separation. At least not for something simple like this.